template/scripts/drupal-vm/drupal-vm.drushrc.aliases.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+// [vagrant_machine_name].local
+$aliases['${project.machine_name}.local'] = array(
```

Comment:
That's often an annoyingly long string, though... it's not a huge deal to me either way, but I like [machinename].local as a convention, and then developers can technically override their local site URLs and use whatever they'd like.

Some devs like "mysitename.local", others "www.mysitename.dev", others "local.mysitename.com", etc.

Having this particular string separate from the URI gives devs a lot more flexibility, imo.